go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -2154,38 +2151,67 @@ func ovnToHostNetworkNormalActionFlows(netConfig *bridgeUDNConfiguration, srcMAC
 		ipFamilyDest = "nw_dst"
 	}
 
+	formatFlow := func(inport, destIP, ctMark string) string {
```

Comment:
you should add a generic comment about what this flow does, and how its priority plays with other flows. Like your previous comment of:

"instead of being output directly to the NIC by the existing flow at prio=100."